sql/updates/world/2015_XX_YY_ZZ_world.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE `world`.`creature_template` SET `flags_extra`='513' WHERE `entry`='16028';
+INSERT INTO `world`.`spell_script_names` (`spell_id`, `ScriptName`) VALUES ('49575', 'spell_dk_death_grip_pull');
```

Comment:
- Remove hardcoded DB names
- Remove ' around numbers
- Add a DELETE FROM to delete the spell that you inserting.